concepts/conceptpower.py

```diff
@@ -0,0 +1,87 @@
+import requests,json
+import xml.etree.ElementTree as ET
+from requests.auth import HTTPBasicAuth
+
+
+class Conceptpower:
+    # Default behavior is to leave these to the constructor.
+    endpoint = None
+    namespace = None
+
+    def __init__(self, **kwargs):
+        # Give first priority to the class definition, if endpoint or namespace are defined (and not None).
+        if self.endpoint is None:
+            self.endpoint = kwargs.get(
+                "endpoint", "http://chps.asu.edu/conceptpower/rest/")
```

Comment:
these should not be hard-coded; if a default is needed it should come from environment variables/settings.